mycore-pi/src/main/resources/catalog.xml

```diff
@@ -107,7 +107,7 @@
   <!-- Crossref 4.4.1 -->
   <system systemId="http://data.crossref.org/schemas/crossref4.4.1.xsd" uri="xsd/crossref/4.4.1/crossref4.4.1.xsd" />
   <system systemId="http://data.crossref.org/schemas/common4.4.1.xsd" uri="xsd/crossref/4.4.1/common4.4.1.xsd" />
-  <system systemId="http://data.crossref.org/schemas/common4.3.5.xsd" uri="xsd/crossref/4.3.5/common4.3.5.xsd" />
+  <system systemId="http://data.crossref.org/schemas/common4.3.5.xsd" uri="xsd/crossref/4.4.1/common4.3.5.xsd" />
```

Comment:
URI seems odd with 4.4.1 and 4.3.5

Reply:
That's O.K the old version xsd. lies in the folder of the new one